x-pack/legacy/plugins/monitoring/public/lib/setup_mode.js

```diff
@@ -83,7 +86,7 @@ export const updateSetupModeData = async (uuid, fetchWithoutClusterUuid = false)
   const data = await fetchCollectionData(uuid, fetchWithoutClusterUuid);
   setupModeState.data = data;
 
-  const isCloud = chrome.getInjected('isOnCloud');
+  const isCloud = core.injectedMetadata.getInjectedVar('isOnCloud');
```

Comment:
also should migrate from `getInjectedVar ` to the client config in the new platform plugin